### ISSUE TITLE
Python 3?

### ISSUE THREAD

**Issue by lukebarone** [llSourcell/predicting_stock_prices#13]
Does this code support Python3? I'm running on WSL (Ubuntu), and it is stalling at the `svr_poly.fit(dates,prices)` line.

It has been sitting here for 4 hours. I have also tried the other issues here, such as chaning the first line in the `predict_prices` function to be `dates = np.reshape(dates, (-1, 1))`. I inserted print statements after each line, and that's how I know which line it's holding up on.

**Comment by Mohswalm545** [comment]
yes it works for me on python3 and thats not a good way to know witch line its holding on
